package.json

```diff
@@ -1,38 +1,47 @@
 {
   "name": "trackovid19-backend",
   "version": "1.0.0",
-  "description": "",
-  "main": "server.js",
+  "description": "Trackovid-19 API",
+  "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js",
-    "start:dev": "nodemon server.js"
+    "start": "node app.js",
+    "dev": "nodemon -w debug -P app.js",
```

Comment:
Nodemon is missing on the dev dependencies

Reply:
Sure why not?